packages/checkout/README.md

```diff
@@ -1,123 +1,182 @@
-<div align="center">
-  <p align="center">
-    <a href="https://docs.x.immutable.com/docs">
-      <img src="https://cdn.dribbble.com/users/1299339/screenshots/7133657/media/837237d447d36581ebd59ec36d30daea.gif" width="280"/>
-    </a>
-  </p>
-  <h1>Immutable Checkout SDK</h1>
-</div>
+# Checkout
 
-# Overview
+## Table of Contents
 
-> **Warning** **IMMUTABLE CHECKOUT SDK IS UNSTABLE** <br/>
-> Since it has not hit the version 1.0 yet, its public interface should not be considered final. Future releases may include breaking changes without further notice. We will do our best to keep this documentation updated providing visibility on breaking changes planned.
+- [Pre-requisites](#pre-requisites)
+- [Checkout SDK Quick Start Guide](#checkout-sdk-quick-start-guide)
+- [Widgets Library Quick Start Guide](#widgets-library-quick-start-guide)
+  - [Widgets Development Locally](#widgets-development-locally)
+- [Project structure](#project-structure)
 
-The Immutable Checkout SDK provides both programatical and user interfaces for interacting with various wallets and L2 networks. It provides all the tools to enable transaction ready gamers including on ramping, dex swaps, bridging and more.
+### Pre-requisites
 
-## Documentation
+Install dependencies for the workspace
 
-- Coming Soon &trade;
-
-# Project structure
-
-```
-/
-│
-└───sdk -> Checkout SDK
-│
-└───sdk-sample-app --> Sample app where the Checkout SDK is consumed
-│
-└───widgets-lib --> Checkout widgets
-│
-└───widgets-sample-app --> Sample app where the Checkout Widgets are consumed
-│
-└───README.md
+```bash
+yarn install
 ```
 
-# Running locally
+### Checkout SDK Quick Start Guide
 
-## Dependencies
+All commands below need to be run in the context of the `checkout-sdk` package unless specified otherwise. Read more about context [here](../../../README.md#context).
```

Comment:
did you mean this is `checkout/sdk` package or is there another package called `checkout-sdk`?

Reply:
The package names are used here instead of the directories. I've added the extra text at the end, for eg: "the context of the `checkout-sdk` package in `/packages/checkout/sdk`" to specify the folder too for that context sentence. I've made some changes throughout the documentation to be more clear, shouldn't be confusing anymore hopefully 😄